src/Commands/Backup/GetCommand.php

```diff
@@ -37,14 +33,14 @@ class GetCommand extends TerminusCommand implements SiteAwareInterface, RequestA
      * @usage <site>.<env> --to=<path> Saves the most recent backup of any type in <site>'s <env> environment to <path>.
      * @usage <site>.<env> --to=<path> Saves the most recent <element> backup in <site>'s <env> environment to <path>.
      */
-    public function getBackup($site_env, array $options = ['file' => null, 'element' => null, 'to' => null,])
+    public function getBackup($site_env, array $options = ['file' => null, 'element' => 'all', 'to' => null,])
```

Comment:
@alexfornuto By using the `'element' => 'all'` here, the underlying software we're using picks it up and adds it to the help output.
![image](https://cloud.githubusercontent.com/assets/868204/22384719/620a3224-e484-11e6-8ed1-95849c83d379.png)